packages/rocketchat-livechat/app/client/lib/LivechatVideoCall.js

```diff
@@ -0,0 +1,80 @@
+/* globals LivechatVideoCall, cordova, JitsiMeetExternalAPI */
+
+LivechatVideoCall = new (class LivechatVideoCall {
+	constructor() {
+		this.live = new ReactiveVar(false);
+		this.calling = new ReactiveVar(false);
+
+		if (typeof JitsiMeetExternalAPI === 'undefined') {
+			$.getScript('/packages/rocketchat_videobridge/client/public/external_api.js');
+		}
+	}
+
+	askPermissions(callback) {
+		if (Meteor.isCordova) {
+			cordova.plugins.diagnostic.requestCameraAuthorization(() => {
+				cordova.plugins.diagnostic.requestMicrophoneAuthorization(() => {
```

Comment:
hmm.. actually, this code does not run on our mobile apps (there is no way to access the route `/livechat` from within the app).

so yeah, this code is useless =)